UKHomeOffice/sgmr-service
# SPCR-387 save and exit

### AC / Description of changes
Save and continue takes user to the responsible people page
Save and come back later takes user to the dashboard

No changes are made to the voyage on these button clicks as the only change to the voyage that can be made on this page is to remove a person, and that is PATCHed to the voyage when remove it clicked


### To Test

- create a voyage up to manifest page
- click save and come back later
- > you should be taken to the dashboard
- return to the manifest page
- > it should be the same as before
- click save and continue
- > you should be taken to the responsible person page


### Notes

---

**Remember to merge to feature branches not `master`.**
